src/utils/ColorUtils.js

```diff
@@ -37,7 +37,7 @@ define(function (require, exports, module) {
      * or color name format according CSS Color Module Level 3 (http://www.w3.org/TR/css3-color/).
```

Comment:
`rebeccapurple` is already CSS Color Module Level 4 (Source: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords)). It's probably not the only change, though, so I don't know what to do with this comment.